### PR TITLE
Skeleton for SWESARR

### PR DESCRIPTION
**Pull request by drboyd**
Hi everyone.  This is a skeleton for SWESARR's tutorial page.

* I need to get access to both the main github repository for editing and the cloud computing resources to test this build for the website. I've tested this on my local computer, and I think it should work fine considering how simple it is.

* I placed this tutorial under "day 3" following the format of 2021's hackweek. It doesn't seem like there's a set table-of-contents structure right now, so please reject this if this goes against the intended table-of-contents structure for 2022.

thanks!

### CODE REVIEW

**Comment by drboyd** [comment]
I apologize. I was looking at a local installation of last year's table of contents. It looks like it was updated!
https://snowex-hackweek.github.io/website/intro.html

Either way, I would like to group the SWESARR tutorial with other groups. I considered putting it under "NASA Missions", "Microwave Missions", "SWE Missions", "Airborne Missions", etc. but I don't think that's a decision I should make. That's why I struck with "Day 3."

**Comment by scottyhq** [comment]
thanks for opening this @db1950 

> I need to get access to both the main github repository for editing and the cloud computing resources to test this build  

Sorry! not sure why you weren't already added there, should be good to go now. 

> Either way, I would like to group the SWESARR tutorial with other groups I considered putting it under "NASA Missions", "Microwave Missions", "SWE Missions", "Airborne Missions".  

@jomey any preference? Whatever you think is best. Can always reorganize later!

File: book/_toc.yml [hunk 1]
```diff
@@ -46,6 +46,11 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+        - file: tutorials/day_3
+          sections:
+            - file: tutorials/microwave
+              sections:
+                - file: tutorials/microwave/swesarr
```
Comment (scottyhq) [inline book/_toc.yml]:
```suggestion
            - file: tutorials/microwave/index
              sections:
                - file: tutorials/microwave/swesarr/index
                - file: tutorials/microwave/swesarr/swesarr_tut
```

**Comment by jomey** [comment]
> @jomey any preference? Whatever you think is best. Can always reorganize later!

My vote would be a specific group like "Microwave Missions" and agree that it can be changed later (and probably will as we shake things out).

**Review by scottyhq — CHANGES_REQUESTED** [review]
@db1950 you can either commit the above 'suggested' changes, or modify the folder structure in the table of contents another way

**Comment by drboyd** [comment]
I'm not sure why the add-preview is failing here, and the error generated isn't making sense to me?

File: book/tutorials/day_3.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Day 3
+
+```{tableofcontents}
+```
```
Comment (jomey) [inline book/tutorials/day_3.md:4]: Looks like this file snuck in when porting the notebook from last year. Should be safe to remove

File: book/_toc.yml [hunk 3]
```diff
@@ -46,6 +46,14 @@ parts:
...
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+        - file: tutorials/microwave
+          title: Microwave Tutorials
+          sections:
+            - file: tutorials/microwave/index
+              sections:
+                - file: tutorials/microwave/swesarr/index
+                - file: tutorials/microwave/swesarr/swesarr_tut
+                  title: SWESARR
```
Comment (jomey) [inline book/_toc.yml]: I think the TOC entry should look like this to make the preview happy:
```
        - file: tutorials/microwave/index
          title: Microwave Tutorials
          sections:
            - file: tutorials/microwave/swesarr/index
            sections:
              - file: tutorials/microwave/swesarr/swesarr_tut
                title: SWESARR
```

**Comment by drboyd** [comment]
Is this file structure not possible on the website? I can't see why this is failing.

**Comment by jomey** [comment]
Dylan, note my PR #74 to suggest your desired structure as I understood it.

**Comment by drboyd** [comment]
@jomey resolved all isues in pull request #74, so we can wrap this one up. Thanks, @jomey!!!

**Pull request closed without merging**